core/Cargo.toml

```diff
@@ -47,6 +47,7 @@ enumset = "1.0.12"
 static_assertions = "1.1.0"
 rustversion = "1.0.11"
 bytemuck = "1.13.0"
+clap = { version = "4.1.4", features = ["derive"] }
```

Comment:
This should be optional and then enabled by a festure that desktop uses. I think we do this in wgpu create so you can see how it works there